THEMES.md

```diff
@@ -713,7 +740,7 @@ Can be created as an extra.
 * `fontPath` - type: PATH.
 * `fontSize` - type: FLOAT.
 * `scrollSound` - type: PATH.
-	- Sound that is played when the list is scrolled.
+	- **Deprecated**: Use global property `sound name="scroll"` instead. Sound that is played when the list is scrolled.
```

Comment:
- I read the code. This still works, which is a good choice, but I imagine that the majority of themes use this, so I'd rather not mark this as deprecated but rather as a fallback option (which seems to be what the code uses it for).

Reply:
New themes should use global property `sound name="scroll"`, "scrollSound" can still be used for backward compatibility. Maybe "deprecated" is not the right word, if you have a better suggestion I will take it.

Reply:
It is a fallback, I suppose - if you don't have a global setting defined for the system, it will use this one. I like the way you explain it - it can still be used for backward compatibility.

I don't have a strong opinion, but I'm mindful of there existing hundreds of themes right now, potentially using the previous syntax, and on the flipside, not wanting to risk this being effectively removed at some point.